privacyidea/api/lib/prepolicy.py

```diff
@@ -541,7 +545,19 @@ def init_tokenlabel(request=None, action=None):
                                                     allow_white_space_in_action=True,
                                                     audit_data=g.audit_object.audit_data)
     if len(imageurl_pols) == 1:
-        request.all_data["appimageurl"] = list(imageurl_pols)[0]
+        request.all_data[ACTION.APPIMAGEURL] = list(imageurl_pols)[0]
+
+    # check the force_app_pin policy
+    app_pin_pols = policy_object.match_policies(action='{0!s}_force_app_pin'.format(token_type),
+                                                scope=SCOPE.ENROLL,
+                                                user_object=user_object,
+                                                client=g.client_ip,
+                                                active=True,
+                                                audit_data=g.audit_object.audit_data)
+    if app_pin_pols:
+        request.all_data["force_app_pin"] = True
+    else:
+        request.all_data["force_app_pin"] = False
```

Comment:
Is there a reason this is set to ``False``?  I think if it was not set, we would have the same outcome.